package.json

```
@@ -57,14 +65,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.0",
     "rollup": "^2.70.2",
-    "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-commonjs": "^21.1.0",
-    "@rollup/plugin-image": "^2.0.5",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^13.2.1",
-    "@rollup/plugin-replace": "^2.3.3",
-    "@rollup/plugin-virtual": "^2.0.3",
-    "@rollup/pluginutils": "^4.1.0",
+    "rollup-plugin-node-externals": "^4.0.0",
     "stylelint": "^14.1.0",
     "turbo": "^1.2.8",
     "typescript": ">=3.0.0"
```

This should be a bit nearer to reality rather than "any TS version release from the last 4 years".

```suggestion
    "typescript": "^4.3.5"
```

Run a `yarn` after this